src/components/_inputs/TextField/TextField.styles.ts

```diff
@@ -1,5 +1,37 @@
+import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 
-export const TextInput = styled.input`
+import { sizes } from '@/styles'
+
+type NodeWidth = { leftNodeWidth: number; rightNodeWidth: number }
+export const TextInput = styled.input<NodeWidth>`
   width: 100%;
+  padding-left: ${({ leftNodeWidth }) => (leftNodeWidth ? `${leftNodeWidth}px !important` : 'inherit')};
+  padding-right: ${({ rightNodeWidth }) => (rightNodeWidth ? `${rightNodeWidth}px !important` : 'inherit')};
+`
+
+export const TextFieldContainer = styled.div`
+  position: relative;
+`
+
+type PlacementProps = {
+  left?: boolean
+}
+const nodePlacementStyles = ({ left }: PlacementProps) =>
+  left
+    ? css`
+        left: 0;
+      `
+    : css`
+        right: 0;
+      `
+export const NodeContainer = styled.div<PlacementProps>`
+  position: absolute;
+  display: flex;
+  align-items: center;
+  height: 50px;
```

Comment:
Where does this come from?

Reply:
I guess you mean the 50px height, it's just what the input height is including the border I just have it fixed for the absolute nodes to work properly, and because this component should not be bigger or smaller than 50px

Reply:
We shouldn't need this height stated here explicitly, this leaves out room for problems if this height changes somehow and we miss it. If this is positioned absolutely, we can just use `bottom: 0` instead